database/src/main/kotlin/no/nav/su/se/bakover/database/person/PersonPostgresRepo.kt

```diff
@@ -0,0 +1,70 @@
+package no.nav.su.se.bakover.database.person
+
+import no.nav.su.se.bakover.common.UUID30
+import no.nav.su.se.bakover.database.hent
+import no.nav.su.se.bakover.database.withSession
+import no.nav.su.se.bakover.domain.Fnr
+import java.util.UUID
+import javax.sql.DataSource
+
+internal class PersonPostgresRepo(
+    private val dataSource: DataSource
+) : PersonRepo {
+    override fun hentFnrForSak(sakId: UUID): Fnr? {
+        return dataSource.withSession { session ->
+            """
+               select fnr
+               from sak
+               where id=:sakId
+|           """
+                .trimMargin()
+                .hent(mapOf("sakId" to sakId), session) {
+                    Fnr(it.string("fnr"))
+                }
+        }
+    }
+
+    override fun hentFnrForSøknad(søknadId: UUID): Fnr? {
+        return dataSource.withSession { session ->
+            """
+               SELECT sak.fnr
+               FROM søknad
+               INNER JOIN sak ON søknad.sakid = sak.id
+               WHERE søknad.id=:søknadId
+            """
+                .trimMargin()
+                .hent(mapOf("søknadId" to søknadId), session) {
+                    Fnr(it.string("fnr"))
+                }
+        }
+    }
+
+    override fun hentFnrForBehandling(behandlingId: UUID): Fnr? {
+        return dataSource.withSession { session ->
+            """
+               SELECT s.fnr
+               FROM behandling b
+               INNER JOIN sak s ON b.sakid = s.id
+               WHERE b.id=:behandlingId
+            """
+                .trimMargin()
+                .hent(mapOf("behandlingId" to behandlingId), session) {
+                    Fnr(it.string("fnr"))
+                }
+        }
+    }
+
+    override fun hentFnrForUtbetaling(utbetalingId: UUID30): Fnr? {
+        return dataSource.withSession { session ->
+            """
+               SELECT fnr
+               FROM utbetaling
+               WHERE id=:utbetalingId
```

Comment:
Kan vi joine her og? Jeg tror tanken med å ha fnr duplikat på utbetaling er for å lagre det vi sendte til oppdrag, slik at dersom en person endrer fnr (og vi endrer det i sak), vil det ikke endres her. Hvis vi kommer til å endre det begge steder, burde vi bare slette fnr fra utbetaling.

@jacob-meidell